<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KittySDK::GGEditWindow</class>
 <widget class="QWidget" name="KittySDK::GGEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::FieldsStayAtSizeHint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="uinLabel">
     <property name="text">
      <string>UIN:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="uinLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="passwordLineEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="ServersGroupBox">
     <property name="title">
      <string>Servers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="serversLabel">
        <property name="text">
         <string>If connection to HUB fails, we will try to connect to one of these servers.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="serversTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
